theano/gpuarray/dnn.py

```diff
@@ -116,9 +116,6 @@ def _dnn_check_compile():
         params.extend(['-I%s%s%s' % (path_wrapper, config.dnn.include_path, path_wrapper)])
     if config.dnn.library_path:
         params.extend(['-L%s%s%s' % (path_wrapper, config.dnn.library_path, path_wrapper)])
-    if config.nvcc.compiler_bindir:
```

Comment:
Can you add a comment why it should not be there? I was pretty sure it needed to be there, so it will help prevent it getting added again.